config/src/iroha.rs

```diff
@@ -120,8 +120,9 @@ impl ConfigurationProxy {
                         ),
                     ));
                 } else {
+                    // TODO: should we just warn the user that this value will be ignored?
```

Comment:
Or rather, fail only if it's not equivalent to the one provided in upmost config? That could be a valid option for the above failure points too.